packages/lib/utils/bytes.ts

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/prefer-default-export
+export const bytesToHuman = (bytes: number) => {
+	const units = ['Bytes', 'KB', 'MB', 'GB'];
+	let unitIndex = 0;
+
+	while (bytes >= 1024 && unitIndex < units.length - 1) {
+		bytes /= 1024;
+		unitIndex++;
+	}
+
+	return `${bytes.toFixed(1)} ${units[unitIndex]}`;
+};
```

Comment:
Hmm, but I've suggested moving it to `@joplin/utils` so why is it in `@joplin/lib` now? /lib should be mostly for business logic, specific to Joplin. /utils is for generic utilities with no dependencies that can be used anywhere. This is so other packages don't need to import the whole lib, renderer, editor and more when all they need is some simple utilities.

Reply:
Ah ok, my bad, I'm fixing this.